_posts/en/newsletters/2023-01-25-newsletter.md

```diff
@@ -0,0 +1,115 @@
+---
+title: 'Bitcoin Optech Newsletter #235'
+permalink: /en/newsletters/2023/01/25/
+name: 2023-01-25-newsletter
+slug: 2023-01-25-newsletter
+type: newsletter
+layout: newsletter
+lang: en
+---
+This week's newsletter summarizes an analysis comparing proposals for
+ephemeral anchors to `SIGHASH_GROUP` and relays a request for
+researchers to investigate how to create proof that an LN async payment
+was accepted.  Also included are our regular sections with summaries of
+popular questions and answers on the Bitcoin Stack Exchange,
+announcements of new releases and release candidates, and descriptions
+of notable changes to popular Bitcoin infrastructure software.
+
+## News
+
+- **Ephemeral anchors compared to `SIGHASH_GROUP`:** Anthony Towns
+  [posted][towns e-vs-shg] to the Bitcoin-Dev mailing list an analysis
+  comparing the recent [ephemeral anchors][topic ephemeral anchors]
+  proposal to an older [`SIGHASH_GROUP` proposal][].  `SIGHASH_GROUP`
+  allows an input to specify which outputs it authorizes, with
+  different inputs in a transaction being able to specify different
+  outputs as long as they don't overlap.  This can be especially useful
+  for adding fees to transactions in contract protocols where two or
+  more inputs are used with presigned transactions.  The presigned
+  nature of these transactions implies that fees may need to be added
+  later when an appropriate feerate is known, and the existing
+  `SIGHASH_ANYONECANPAY` and `SIGHASH_SINGLE` sighash flags aren't
+  flexible enough for multi-input transactions because they only commit
+  to a single input or output.
+
+    Ephemeral anchors, similar to [fee sponsorship][topic fee
+    sponsorship], lets anyone [CPFP][topic cpfp] fee bump a transaction.
+    The transaction being fee bumped is allowed to contain zero fees.
+    Because anyone can fee bump a transaction using ephemeral anchors,
+    this mechanism can also be used to pay fees for the multi-input
+    presigned transactions which are a target for `SIGHASH_GROUP`.
+
+    `SIGHASH_GROUP` would still have two advantages: first, that it
+    could allow [batching][topic payment batching] multiple unrelated
+    presigned transactions, which could reduce transaction size
+    overhead, which would reduce user costs and increase network
+    capacity.  Second, that it doesn't require a child transaction,
+    which would further reduce costs and increase capacity.
+
+    Towns concludes by noting that ephemeral anchors, with its
+    dependency of [v3 transaction relay][topic v3 transaction relay],
+    capture most of the benefits of `SIGHASH_GROUP` and provides the
```

Comment:
I'd say your first suggestion is correct since the subject is singular -- "ephemeral anchors", even though it ends in "s", is singular because it's a concept. I considered suggesting "the ephemeral anchors concept" (or "approach") on line 49 but I think that's too wordy.